### PR TITLE
test(e2e): add more cases of multi-signature service

### PR DESCRIPTION
**Pull request by KaoImin**
**What type of PR is this?**
test

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by LycrusHamster** [comment]
@KaoImin  why change defaul 1337 p2p port to 13390?

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (yejiayu) [inline yarn.lock:1]: the `yarn.lock` should not be in root dir

File: devtools/chain/config.toml [hunk 2]
```diff
@@ -13,7 +13,7 @@ maxconn = 25000
 max_payload_size = 1048576
 
 [network]
-listening_address = "0.0.0.0:1337"
+listening_address = "0.0.0.0:13390"
```
Comment (yejiayu) [inline devtools/chain/config.toml:16]: why

File: core/mempool/src/lib.rs [hunk 3]
```diff
@@ -419,7 +419,7 @@ pub enum MemPoolError {
     InsertCandidate { len: usize },
 
     #[display(fmt = "Tx: {:?} check_sig failed", tx_hash)]
-    CheckSig { tx_hash: Hash },
+    CheckAuthorization { tx_hash: Hash },
```
Comment (yejiayu) [inline core/mempool/src/lib.rs:422]: if you change to `CheckAuthorization` that error message should be changed as well.

File: devtools/chain/config.toml (hunk 2 above)
Comment (KaoImin) [reply devtools/chain/config.toml:16]: The "0.0.0.0:1337" is always occupied.

**Pull request merged**